src/ansari/ansari_db.py

```diff
@@ -109,13 +112,95 @@ def _get_token_from_request(self, request: Request) -> str:
                 detail="Authorization header is malformed",
             )
 
+    def _execute_query(
```

Comment:
You realize we are taking baby steps towards building an ORM, right ? :-)

Wonder if we should just fully commit to an ORM.

Reply:
Totally slipped my mind, I don't know how!

I don't mind later changing this code to support ORM ... my main motivation on why I created this method, was that I found duplications in other methods, so I wanted to refactor, + add logs ... but if we're headed towards ORM, no problems at all 👍